tools/vscode-extension/package.json

```diff
@@ -0,0 +1,131 @@
+{
+  "name": "oss-fuzz",
+  "displayName": "OssFuzz",
+  "description": "",
+  "version": "0.0.2",
+  "publisher": "OSS-Fuzz maintainers",
+  "engines": {
+    "vscode": "^1.76.0"
+  },
+  "categories": [
+    "Other"
+  ],
+  "extensionDependencies": [
+    "mindaro-dev.file-downloader"
+  ],
+  "activationEvents": [],
+  "main": "./build/src/extension.js",
+  "contributes": {
+    "configurations": [
+      {
+        "id": "OSS-Fuzz-Base",
+        "title": "Base path to OSS-Fuzz",
+        "properties": {
+          "oss-fuzz.OSS.Path": {
+            "type": "string"
+          }
+        }
+      }
+    ],
+    "viewsWelcome": [
+      {
+        "view": "workbench.explorer.emptyView",
+        "contents": "You can have paragraphs of text here. You can have [links](https://code.visualstudio.com) to external sources or [internal commands](command:welcome-view-content-sample.hello).\nUse new lines to have new paragraphs.\nPlace a link alone in a paragraph to make it a button\n[Hello](command:welcome-view-content-sample.hello)"
+      }
+    ],
+    "commands": [
+      {
+        "command": "oss-fuzz.SetUpOssFuzz",
```

Comment:
nit: would oss-fuzz.SetUp suffice here to avoid some stutter?

Reply:
Good catch, done!